lib/KTable/KTable.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+
+  <div class="k-table">
+    <table>
+      <caption v-if="caption">
+        {{ caption }}
+      </caption>
+      <thead>
+        <tr>
+          <th
+            v-for="(header, index) in headers"
+            :key="index"
+            tabindex="0"
+            @click="handleSort(index)"
+            @keydown="handleKeydown($event, -1, index)"
+          >
+            {{ header.label }}
+            <span v-if="header.dataType !== 'others'" class="sort-icon">
+              <span v-if="sortKey === index && sortOrder === 'asc'">⬆️</span>
+              <span v-else-if="sortKey === index && sortOrder === 'desc'">⬇️</span>
+              <span v-else>⬍</span>
+            </span>
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="(row, rowIndex) in sortedRows" :key="rowIndex">
+          <KTableGridItem
+            v-for="(cell, cellIndex) in row"
+            :key="cellIndex"
+            :content="cell"
+            :align="headers[cellIndex].dataType === 'numeric' ? 'right' : 'left'"
+            :rowIndex="rowIndex"
+            :cellIndex="cellIndex"
+            @keydown="handleKeydown($event, rowIndex, cellIndex)"
+          />
+        </tr>
+      </tbody>
+    </table>
+  </div>
+
+</template>
+
+
+<script>
+
+  import KTableGridItem from './KTableGridItem.vue';
+
+  export default {
+    name: 'KTable',
+    components: {
+      KTableGridItem,
+    },
+    props: {
+      headers: {
+        type: Array,
+        required: true,
+      },
+      rows: {
+        type: Array,
+        required: true,
+      },
+      caption: {
+        type: String,
+        required: false,
+        default: null,
+      },
+    },
+    data() {
+      return {
+        sortKey: null,
+        sortOrder: 'asc',
```

Comment:
Could be helpful to use constants for `'asc'`, `'desc'`, `'numeric'`, `'date'` etc. strings. You could even define them in the `useTable` composable and import in the component from there. This will help to have them available in both places (and even be imported for custom sort functions in the future)